### ISSUE TITLE
Suggestion: automate this with GitHub Actions

### ISSUE THREAD

**Issue by simonw** [COVID19Tracking/covid-tracking-data#21]
Is this update script automated at the moment or is it being run manually? From the commit history it looks like it might be manual.

I've run several projects similar to this using automated scripts running in Circle CI or (more recently) GitHub Actions. I would be happy to submit a pull request to configure hourly / daily scheduled task in Actions if that would be useful.

**Comment by Nosferican** [comment]
From my understanding the data is in a Google spreadsheet and the API uses that to create the GitHub data backups. I needed to perform some cleaning on the data and and keep it current so, I set up https://github.com/uva-bi-sdad/COVID_Tracking_Clean. Feel free use any of the code to improve the project. It uses GH Actions with a CRON to get the latest data, perform the cleaning, and publish it to the repo.
